.github/workflows/staticbuild.yml

```diff
@@ -76,7 +76,7 @@ jobs:
     - name: Build (Latest)
       if: inputs.release_type == 'latest'
       run: |
-        sudo -Hu testbot bash -lc 'make BUILD_CHANNEL="latest" static-release'
+        sudo -Hu testbot bash -lc 'make RELEASE_TYPE="latest" BUILD_CHANNEL="${{date +%Y%m%d%H%M%S}}" static-release'
```

Comment:
So I'm worried that the date here isn't going to match between different architectures. I'm not sure if that matters or not, but as arm64 and amd64 would be two different jobs, they'd run at slightly different times and thus it wouldn't be clear if/when things were the same "build" or not.

Ex: 
```
viam-server-20240410121314-arm64
viam-server-20240410121318-amd64
```

I think the correct answer would be have have a timestamp input to this workflow, so they'd both use the same one.